example/src/main/java/org/secondbase/example/main/HelloAll.java

```diff
@@ -0,0 +1,49 @@
+package org.secondbase.example.main;
+
+import io.prometheus.client.Counter;
+import java.io.IOException;
+import org.secondbase.consul.ConsulModule;
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.SecondBaseException;
+import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
+import org.secondbase.flags.Flags;
+import org.secondbase.webconsole.HttpWebConsole;
+import org.secondbase.webconsole.PrometheusWebConsole;
+import org.secondbase.webconsole.widget.Widget;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author acidmoose
+ */
+public class HelloAll {
+    @Flag(name="variable")
+    private static String var = "default";
+    @Flag(name="counter")
+    private static int counter = 1;
+
+    private static final Logger LOG = LoggerFactory.getLogger(HelloAll.class.getName());
+
+    public static void main(final String[] args) throws SecondBaseException, IOException {
+        final Counter mycounter = Counter.build("mycounter", "a counter").register();
+
+        final Widget prometheusWidget = new PrometheusWebConsole();
```

Comment:
actually, I can't use the interface if I need to use the methods in the class. For instance, registering a service in consul.